ci/mkpipeline.py

```diff
@@ -218,11 +298,11 @@ def set_default_agents_queue(pipeline: Any) -> None:
             and "group" not in step
             and "trigger" not in step
         ):
-            step["agents"] = {"queue": "linux-aarch64-small"}
+            step["agents"] = {"queue": DEFAULT_AGENT}
 
 
 def check_depends_on(pipeline: Any, pipeline_name: str) -> None:
-    if pipeline_name != "test":
+    if pipeline_name not in ("test", "nightly", "release-qualification"):
```

Comment:
How were these pipelines chosen? I assume these are the testing pipelines. Should we add the SLT pipeline as well?

Reply:
Should we extract this set of values to a named constant in `buildkite_constants.py`? (optional)
(We can also move that file to a more generic place.)

Reply:
These are the pipelines which don't have a global `wait` block after the compilation, since there are jobs that can be run without waiting for all compilations to be done, for example triggering another pipeline, or running an x86-64 job before the arm build is finished.